app/graphql/users/resolvers.js

```diff
@@ -0,0 +1,35 @@
+const { user: User } = require('../../models'),
+  logger = require('../../logger');
+
+exports.createUser = ({ email, password, name, lastName }) => {
+  try {
```

Comment:
change this function concept. Split the validations in multiple small functions and add errors with message and internal code to an array.